dd-sdk-android-gradle-plugin/src/main/kotlin/com/datadog/gradle/plugin/InstrumentationMode.kt

```diff
@@ -0,0 +1,39 @@
+package com.datadog.gradle.plugin
+
+/**
+ * Defines the mode of instrumentation for the Kotlin compiler plugin.
```

Comment:
```suggestion
 * Defines the mode of instrumentation for the Kotlin Compiler plugin.
```

but maybe we should remove the mention of `Kotlin Compiler` plugin here completely, because it is implementation details. And maybe just say "plugin"? Same applies to the docs below.